jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/GeoJsonConverterProperties.java

```diff
@@ -119,6 +119,10 @@ public Properties createStreamProperties(String name) {
         // Configure the state store location
         streamProps.put(StreamsConfig.STATE_DIR_CONFIG, "/var/lib/ode/kafka-streams");
 
+        streamProps.put(ProducerConfig.COMPRESSION_TYPE_CONFIG, "zstd");
+
+        streamProps.put(ProducerConfig.LINGER_MS_CONFIG, 50);
```

Comment:
Do you think we should make  the compression type and/or linger configurable in .env/application.yaml?